src/pages/CustomerRequisitionPage.js

```diff
@@ -91,9 +89,24 @@ export const CustomerRequisition = ({
   onEditCreatedDate,
   datePickerIsOpen,
   onDatePickerClosed,
+  isRemoteOrder,
+  columnSet,
+  indicatorColumns,
 }) => {
   const { isFinalised, comment, program, isResponse, createdDate, entryDate } = pageObject;
 
+  const currentColumns = useMemo(() => {
+    const formEntryColumnSet = isRemoteOrder
+      ? 'editableCustomerRequisitionFormEntry'
+      : 'customerRequisitionFormEntry';
+    if (usingIndicators && showIndicators) {
+      return indicatorColumns;
+    }
+    return columnSet === 'a'
```

Comment:
Any particular reasoning for this literal? Could perhaps benefit from a constant (or a comment, in the meantime)?

Reply:
it's just the state that's set, not sure what comment I could write that would help

Reply:
Is there a reason the `columnSet` is `'a'` rather than a constant? Does the literal have an obvious meaning that I'm missing?

Reply:
No, the action/reducer switches between "a" and "b" (This is just taken from the `mapStateToProps`)